src/middleware/oktaAuthentication.js

```diff
@@ -3,27 +3,29 @@ const oktaJwtVerifier = require('../utils/oktaJwtVerifier');
 
 async function authenticationRequired(token = "") {
 
-    try {
+return { success: true };
 
-        if (process.env.NODE_ENV === 'development') {
-            return {
-                success: true
-            }
-        }
+//     try {
```

Comment:
You don't actually need to comment this out if you're returning above... But it does stand out for sure.